max72xx/max72xx.go

```diff
@@ -0,0 +1,96 @@
+// Driver works for max7219 and 7221
+// Datasheet: https://datasheets.maximintegrated.com/en/ds/MAX7219-MAX7221.pdf
+package max72xx
+
+import (
+	"machine"
+)
+
+type Device interface {
```

Comment:
Not sure why this is declared as an interface here. Seems like simpler to just implement directly without the indirection.

Reply:
removed it